### PR TITLE
Convert ResNet to SwiftPM and add notes how to run it with Py3.

### PR DESCRIPTION
**Pull request by xiejw**
In addition to the changes mentioned in the title, I did the same change for CIFAR regarding the py3 issue.

### CODE REVIEW

File: CIFAR/README.md [hunk 1]
```diff
@@ -5,13 +5,18 @@ classification on the [CIFAR-10](https://www.cs.toronto.edu/~kriz/cifar.html) da
...
 ## Setup
 
-You'll need the [latest version of Swift for TensorFlow](https://github.com/tensorflow/swift/blob/master/Installation.md)
+You'll need the latest version of [Swift for TensorFlow][SwiftForTensorFlow]
 installed and added to your path. Additionally, the data loader requires Python
-3.x (rather than Python 2.7), `wget`, and numpy.
+3.x (rather than Python 2.7), `wget`, and `numpy`.
+
+> Note: For MacOS, you need to set up the `PYTHON_LIBRARY` to help the Swift for
+> TensorFlow find the `libpython3.<minor-value>.dylib` file, e.g., in `homebrew`.
```
Comment (rxwei) [inline CIFAR/README.md]: Minor value -> minor version.

File: CIFAR/README.md [hunk 2]
```diff
@@ -5,13 +5,18 @@ classification on the [CIFAR-10](https://www.cs.toronto.edu/~kriz/cifar.html) da
 
 ## Setup
 
-You'll need the [latest version of Swift for TensorFlow](https://github.com/tensorflow/swift/blob/master/Installation.md)
+You'll need the latest version of [Swift for TensorFlow][SwiftForTensorFlow]
 installed and added to your path. Additionally, the data loader requires Python
-3.x (rather than Python 2.7), `wget`, and numpy.
+3.x (rather than Python 2.7), `wget`, and `numpy`.
+
+> Note: For MacOS, you need to set up the `PYTHON_LIBRARY` to help the Swift for
```
Comment (rxwei) [inline CIFAR/README.md]: MacOS -> macOS

File: CIFAR/README.md [hunk 3]
```diff
@@ -5,13 +5,18 @@ classification on the [CIFAR-10](https://www.cs.toronto.edu/~kriz/cifar.html) da
 
 ## Setup
 
-You'll need the [latest version of Swift for TensorFlow](https://github.com/tensorflow/swift/blob/master/Installation.md)
+You'll need the latest version of [Swift for TensorFlow][SwiftForTensorFlow]
```
Comment (rxwei) [inline CIFAR/README.md]: It’d be better to put the link on “the latest version” instead of the name of the product.

File: ResNet/README.md [hunk 4]
```diff
@@ -6,13 +6,18 @@ dataset.
 
 ## Setup
 
-You'll need the [latest version of Swift for TensorFlow]
-(https://github.com/tensorflow/swift/blob/master/Installation.md)
-installed and added to your path. Additionally, the data loader requires Python 3.x (rather than
-Python 2.7), `wget`, and numpy.
+You'll need the latest version of [Swift for TensorFlow][SwiftForTensorFlow]
```
Comment (rxwei) [inline ResNet/README.md]: Same here.

File: CIFAR/README.md (hunk 1 above)
Comment (xiejw) [reply CIFAR/README.md]: Done.

File: CIFAR/README.md (hunk 2 above)
Comment (xiejw) [reply CIFAR/README.md]: Done

File: CIFAR/README.md (hunk 3 above)
Comment (xiejw) [reply CIFAR/README.md]: Done.

File: ResNet/README.md (hunk 4 above)
Comment (xiejw) [reply ResNet/README.md]: Done.

**Comment by xiejw** [comment]
PTAL. Thanks

**Pull request merged**